components/Transactions.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { TouchableOpacity } from 'react-native';
+
+import {
+  Card,
+  ContentContainer,
+  IconContainer,
+  MainText,
+  Overline
+} from '../styles/transactions';
+
+/**
+ * @prop
+ * */
+
+function Transactions(props) {
```

Comment:
I modified this component to be a function instead -- generally if we don't need state at all and only rely on props, it means we should likely just make this a function. Also because eslint yelled at me